packages/schema-blocks/src/functions/innerBlocksHelper.ts

```diff
@@ -23,16 +22,4 @@ function getInnerblocksByName( blockNames: string[], blockInstances: BlockInstan
 	return foundBlocks;
 }
 
-/**
- * Gets the inner blocks of the block with the given clientId from the core/block-editor store.
- *
- * @param clientId The clientId of the block whose InnerBlocks you want.
- *
- * @returns {BlockInstance[]} The block's inner blocks.
- */
-function getInnerBlocks( clientId: string ): BlockInstance[] {
-	/* istanbul ignore next */
-	return select( "core/block-editor" ).getBlock( clientId ).innerBlocks;
-}
-
-export { getInnerBlocks, getInnerblocksByName };
+export { getInnerblocksByName };
```

Comment:
The brackets are unnecessary here 🙂.
```suggestion
export getInnerblocksByName;
```